### PR TITLE
Chat: Proxy stateEnabled options to button and textarea

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: packages/devextreme/js/__internal/ui/chat/chat.ts [hunk 1]
```diff
@@ -110,6 +119,11 @@ class Chat extends Widget<Properties> {
     const { name, value } = args;
 
     switch (name) {
+      case 'activeStateEnabled':
+      case 'focusStateEnabled':
+      case 'hoverStateEnabled':
+        this._messageBox.option({ [name]: value });
```
Comment (EugeniyKiyashko) [inline packages/devextreme/js/__internal/ui/chat/chat.ts:125]:
```suggestion
        this._messageBox.option(name, value);
```

File: packages/devextreme/js/__internal/ui/chat/chat_message_box.ts [hunk 2]
```diff
@@ -90,9 +109,19 @@ class MessageBox extends Widget<MessageBoxProperties> {
...
-    const { name } = args;
+    const { name, value } = args;
 
     switch (name) {
+      case 'activeStateEnabled':
+      case 'focusStateEnabled':
+      case 'hoverStateEnabled': {
+        const options = { [name]: value };
+
+        this._button.option(options);
```
Comment (EugeniyKiyashko) [inline packages/devextreme/js/__internal/ui/chat/chat_message_box.ts:120]:
```suggestion
        this._button.option(name, value);
```

File: packages/devextreme/js/__internal/ui/chat/chat_message_box.ts [hunk 3]
```diff
@@ -90,9 +109,19 @@ class MessageBox extends Widget<MessageBoxProperties> {
...
+        this._textArea.option(options);
```
Comment (EugeniyKiyashko) [inline packages/devextreme/js/__internal/ui/chat/chat_message_box.ts:121]:
```suggestion
        this._textArea.option(name, value);
```

**Review by EugeniyKiyashko — APPROVED** [review]

File: packages/devextreme/js/__internal/ui/chat/chat.ts (hunk 1 above)
Comment (marker-dao) [reply packages/devextreme/js/__internal/ui/chat/chat.ts:125]: can't cause TS error

File: packages/devextreme/js/__internal/ui/chat/chat_message_box.ts (hunk 2 above)
Comment (marker-dao) [reply packages/devextreme/js/__internal/ui/chat/chat_message_box.ts:120]: can't cause TS error

File: packages/devextreme/js/__internal/ui/chat/chat_message_box.ts (hunk 3 above)
Comment (marker-dao) [reply packages/devextreme/js/__internal/ui/chat/chat_message_box.ts:121]: can't cause TS error

**Pull request merged**